### PR TITLE
Basic Recruit Training Finished

### PR DESCRIPTION
**Pull request by bcayanan**
For this basic recruit training, I have completed what was asked of me within both `person` and `pointers` exercises.

### CODE REVIEW

**Review by JessieG123 — CHANGES_REQUESTED** [review]
Please add a description to the PR. 
~~I think you also need to config email and username since your profile icon doesn't show up with your commit.~~ JK I see it now.

File: Recruit-Training/Basic-Recruit-Training/person/Person.cpp [hunk 1]
```diff
@@ -0,0 +1,57 @@
+#include "Person.h"
+#include <iostream>
+
+using namespace::std;
```
Comment (JessieG123) [inline Recruit-Training/Basic-Recruit-Training/person/Person.cpp:4]: Not sure if you can do that.... but either way if you're going to use it do it like `using namespace std`. Also it is bad practice to use it in general so I wouldn't recommending using it.

File: Recruit-Training/Basic-Recruit-Training/person/Person.cpp [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+	cout << "Name: " << name_ << endl;
+	cout << "Age: " << *age_ << endl;
+}
+
+int Person::combinedAge(Person** personArray, int size)
+{
+	int result = 0;
+
+	for(int i = 0; i < size; i++)
+    {
```
Comment (JessieG123) [inline Recruit-Training/Basic-Recruit-Training/person/Person.cpp:46]: Consistent indentation please.

**Review by JessieG123 — CHANGES_REQUESTED** [review]

**Review by duan-le — APPROVED** [review]
Everything looks good to me. Please **DO NOT** merge this branch into `master`. Get an approval from another reviewer and then you can close this PR.

**Review by JessieG123 — APPROVED** [review]
Looks great! Awesome job Bryce! Now, **do not** merge the PR, instead close it. 
Let us know if you have any questions!

**Pull request closed without merging**